### PR TITLE
Add DAWON DNS / SG-V100-ZB

### PR DESCRIPTION
**Pull request by nukusinji**
Add IoT smart gas lock SG-V100-ZB

### CODE REVIEW

**Comment by Koenkk** [comment]
I see some unexpected changes under https://github.com/Koenkk/zigbee2mqtt.io/pull/657/files , please rebase your pr.

**Comment by nukusinji** [comment]
Rebased~

File: docs/devices/TH1300ZB.md [hunk 1]
```diff
@@ -12,12 +12,11 @@ description: "Integrate your Sinope TH1300ZB via Zigbee2MQTT with whatever smart
 | Model | TH1300ZB  |
 | Vendor  | Sinope  |
 | Description | Zigbee smart floor heating thermostat |
-| Exposes | local_temperature, lock (state), climate (occupied_heating_setpoint, local_temperature, system_mode, running_state, pi_heating_demand), backlight_auto_dim, linkquality |
```
Comment (Koenkk) [inline docs/devices/TH1300ZB.md:15]: Why was this changed?

File: docs/devices/TH1300ZB.md [hunk 2]
```diff
@@ -30,62 +29,62 @@ The device will restart. The wireless icon will disapear from the display.
...
 The thermostat can be instructed to use the occupancy or unoccupancy setpoints by
-publishing to `zigbee2mqtt/FRIENDLY_NAME/set {"thermostat_occupancy": "OCCUPANCY"}`
+publishing to `zigbee2mqtt/[FRIENDLY_NAME]/set {"thermostat_occupancy": "OCCUPANCY"}`
 where `OCCUPANCY` is one of the following values: `occupied`, `unoccupied`.
 The factory default value is `occupied`.
 
 #### Floor control mode
 The sensor used by the thermostat for temperature regulation can be changed by publishing
-to `zigbee2mqtt/FRIENDLY_NAME/set {"floor_control_mode": "SENSOR"}`
+to `zigbee2mqtt/[FRIENDLY_NAME]/set {"floor_control_mode": "SENSOR"}`
```
Comment (Koenkk) [inline docs/devices/TH1300ZB.md:38]: This part of the docs is generated, so please don't add `[` here (as it will be overwitten again).

File: docs/devices/WSP404.md [hunk 3]
```diff
@@ -11,41 +11,14 @@ description: "Integrate your OWON WSP404 via Zigbee2MQTT with whatever smart hom
...
+| Description | Plug-in smart switch |
+| Supports | on/off, power and energy measurement |
 | Picture | ![OWON WSP404](../images/devices/WSP404.jpg) |
 
 ## Notes
 
 None
 
-
-## Exposes
```
Comment (Koenkk) [inline docs/devices/WSP404.md:23]: Why was this changed?

File: docs/devices/TS0121_plug.md [hunk 4]
```diff
@@ -21,53 +21,7 @@ description: "Integrate your TuYa TS0121_plug via Zigbee2MQTT with whatever smar
 ### Device type specific configuration
 *[How to use device type specific configuration](../information/configuration.md)*
 
-* `measurement_poll_interval`: This device does not support reporting electric measurements so it is polled instead. The default poll interval is 60 seconds.
-
-
-## Exposes
-
-### Switch
```
Comment (Koenkk) [inline docs/devices/TS0121_plug.md:29]: Why was this changed?

File: docs/devices/TH1300ZB.md (hunk 1 above)
Comment (nukusinji) [reply docs/devices/TH1300ZB.md:15]: I don't know, Why other files changed...
I only add 2files(SG-V100-ZB.md, SG-V100-ZB.jpg)

**Comment by nukusinji** [comment]
Sorry. I will reopen check my fault.
